influxql/functions.gen.go.tmpl

```diff
@@ -4,12 +4,31 @@ package influxql
 
 // {{$k.Name}}PointAggregator aggregates points to produce a single point.
 type {{$k.Name}}PointAggregator interface {
-	Aggregate(p *{{.Name}}Point)
+	Aggregate(p *{{$k.Name}}Point)
+}
+
+// {{$k.Name}}BulkPointAggregator aggregates multiple points at a time.
+type {{$k.Name}}BulkPointAggregator interface {
```

Comment:
I have to deal with receiving points in stream when they need to be bulk processed and vice versa so  I need each reducer to support both interfaces. As a result I will be combining these interfaces on my end, if you do not combine them here. Is there a reason to separate them here if we plan on adding the Bulk method to all of them anyways?